<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calcolatrice</class>
 <widget class="QDialog" name="Calcolatrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>750</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calcolatrice</string>
  </property>
  <widget class="QTabWidget" name="Grafici">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>741</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-left-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabTriangolo">
    <attribute name="title">
     <string>Triangolo</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="quattroLati">
    <attribute name="title">
     <string>4 Lati</string>
    </attribute>
    <widget class="QTabWidget" name="graficiQuad">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>741</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabQuadrilatero">
      <attribute name="title">
       <string>Quadrilatero</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabRettangolo">
      <attribute name="title">
       <string>Rettangolo</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabQuadrato">
      <attribute name="title">
       <string>Quadrato</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="Titolo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;TeX Gyre Adventor&quot;;
border-color: rgb(190, 0, 0);
color: rgb(220, 0, 0);</string>
   </property>
   <property name="text">
    <string>Calcolatrice Figure Geometriche</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eliminaFigura">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina Figura</string>
   </property>
  </widget>
  <widget class="QLabel" name="Risultato">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(202, 202, 202);

border: 0.5px solid black;

border-radius: 4px;</string>
   </property>
   <property name="text">
    <string>Risultato</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="inserisciFigura">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inserisci Figura</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labRisultati">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);

padding-left: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Somma">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>640</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Somma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eliminaVertice">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>640</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina Vertice</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
